conans/tools.py

```diff
@@ -194,6 +194,11 @@ def __init__(self, *args, **kwargs):
 run_in_windows_bash = tools_win.run_in_windows_bash
 
 
+# from conans.client.tools.settings
+cppstd_minimum_required = tools_settings.cppstd_minimum_required
+valid_minimum_cppstd = tools_settings.valid_minimum_cppstd
+
+
```

Comment:
These lines are not needed either (see comment above)

Reply:
done